Examples/Swift/ViewController.swift

```diff
@@ -360,7 +360,7 @@ extension ViewController: VoiceControllerDelegate {
         print(interruptedInstruction.text, interruptingInstruction.text)
     }
 
-    func voiceController(_ voiceController: RouteVoiceController, willSpeak instruction: SpokenInstruction, routeProgress: RouteProgress) -> SpokenInstruction? {
+    func voiceController(_ voiceController: RouteVoiceController, willSpeak instruction: SpokenInstruction, routeProgress: RouteProgress?) -> SpokenInstruction? {
```

Comment:
Could you please add a CHANGELOG entry for the changed APIs? Both here and for the RouteVoiceController?